scripts/support/create_ge_images/README.rst

```diff
@@ -0,0 +1,162 @@
+Then the testing script is invoked. If it complete without errors, the VM is deleted,
+the ssh-agent is killed and the UUID of the image is printed. The image is
+ready for publication.
+
+Be aware that if the creation script fails, the virtual machine will not be deleted;
+this is useful to debug the problem, but anyway that virtual machine should be
+deleted before trying again. In the same way, if the testing script fails,
+neither the virtual machine nor the release candidate image are deleted.
+
+**Warning: the testing script is executed directly, it is not invoked inside the
+VM machine but in the user account. Therefore, this script is a security risk
+and must be audited before running it. It is important also not use the root
+account to invoke the process. The script cleans the sudo credential before
+invoking the script. As an experimental feature, it is possible to run the
+script inside a VM. This is described in next section.**
+
+Running the testing script in a VM
+----------------------------------
+
+If ``TEST_USING_VM`` is defined and not empty, the testing script is executed
+inside a VM. This is an experimental feature that does not require an extra
+floating IP. It is more secure, but it needs more time to complete. If the
+script fails, maybe a good idea is to check that this experimental feature is
+not the cause of the problem. To repeat the test only, the variable ``TEST_ONLY``
+must be defined and not empty (and of course to use the traditional method,
+``TEST_USING_VM`` must be undefined).
+
+It works by creating a second VM (the tester). Initially the floating IP is assigned to
+this VM and a SSH connection is created, using SSH ControlMaster; this maintains
+a persistent connection that is reused by the subsequent ssh commands. Then the
+IP is assigned to the tested VM; this change does not affect to the already
+created connections. This way is possible to connect to the tester VM via ssh,
+in spite of the floating IP is assigned now to the tested VM.
+
+Publishing the image
+********************
+
+Every GE image have two properties: the NID and the type. The values are
+registered at https://wikis.hi.inet/fi-ware/index.php/Cloud:GE_Identification
+
+The following three situations are possible:
+
+* the image is completely new. Then a new NID must be allocated and registered in
+  the wiki. The type must be the same that the other images in the same chapter.
+* there is an older version of the image and both will coexist, using for example
+  a suffix with the version number. The NID and type must be provided to the
+  script manually but they must be the same values than the other image.
+* the image replaces an older version, using the same name. The old image
+  will be overwritten. The NID and type of the old image will be used
+  automatically.
+
+The script ``locate_images.py`` can be helpful. It shows a list of the images with
+its name, nid and type. It is also useful to see what NIDs are in use, if a new
```

Comment:
I suggest deletion of the content "...if a new one must be used" due to the NID are obtained directly from the node-id in the catalogue.